modules/ding_mobilesearch/ding_mobilesearch.module

```diff
@@ -117,61 +117,174 @@ function _ding_mobilesearch_types() {
   return $types;
 }
 
-
-
 /**
  * Implements hook_mobilesearch_node_export_mapping().
  */
 function ding_mobilesearch_ding_mobilesearch_node_export_mapping($node) {
+  // Map common fields on the node.
+  $mapping = _ding_mobilesearch_map_entity_fields('node', $node->type, $node);
+
   $username = db_select('users', 'u')
     ->fields('u', array('name'))
     ->condition('u.uid', $node->uid)
     ->execute()
     ->fetchField();
```

Comment:
This is not really a the proper way to get the username for the author of a node but I guess that is not part of the scope of this PR 😕 .

Reply:
Tbh this was a case of getting in and out as fast as possible 😄 Not really a fan of working with this module in general.

But I'll happily update this. Using  `format_username()` on a loaded user object should give more correct result.

Reply:
Done.